### PR TITLE
Update delete channel description.

### PR DESCRIPTION
**Pull request by coltoneshaw**
#### Summary
Some confusion came up with a customer about the term soft delete. This means archived within the UI, so changing the lanquage to match what the customer would experience.

### CODE REVIEW

**Comment by coltoneshaw** [comment]
@cwarnermm - Similar request. Needs review here as well. Thanks!

File: v4/source/channels.yaml [hunk 1]
```diff
@@ -628,12 +628,12 @@
         - channels
       summary: Delete a channel
       description: >
-        Soft deletes a channel, by marking the channel as deleted in the
-        database. Soft deleted channels will not be accessible in the user
-        interface. Direct and group message channels cannot be deleted.
+        Archives a channel. This will set the `deleteAt` to the current timestamp in the database. Soft deleted channels may not be accessible in the user
+        interface. They can be viewed and unarchived in the system console > channel based on your license. Direct and group message channels cannot be deleted.
```
Comment (cwarnermm) [inline v4/source/channels.yaml:632]:
```suggestion
        interface. They can be viewed and unarchived in the **System Console > User Management > Channels** based on your license. Direct and group message channels cannot be deleted.
```

Comment (cwarnermm) [reply v4/source/channels.yaml:632]: I'm a bit unclear about the "based on your license" phrase in this description.

**Review by cwarnermm — COMMENTED** [review]
Feedback provided inline and one clarifying question.

File: v4/source/channels.yaml (hunk 1 above)
Comment (coltoneshaw) [reply v4/source/channels.yaml]: To my knowledge, not all license levels are able to do channel management within the system console. With all the license changes, I'm not sure where this line is drawn anymore.

Comment (cwarnermm) [reply v4/source/channels.yaml]: Great point! Channel management is available in [Mattermost Enterprise](https://docs.mattermost.com/configure/configuration-settings.html#channels). I've updated my recommendation to account for this.

**Pull request merged**